utility/types/message.go

```diff
@@ -41,6 +43,8 @@ type Message interface {
 
 	SetSigner(signer []byte)
 	ValidateBasic() Error
+	// Get the canonical byte representation of the ProtoMsg.
```

Comment:
This comment isn't telling me much more than the function name `GetCanonicalBytes` so let's remove it.
```suggestion
```